src/QAT/qat/purr/backends/verification.py

```diff
@@ -89,25 +93,38 @@ def get_verification_model(qpu_type: QPUVersion) -> Optional[VerificationModel]:
         )
 
     if qpu_type.make == Lucy.__name__:
-        # TODO: Should have an apply_setup_to_hardware in live which people can use to build our own architecture.
-        model = VerificationModel(qpu_type, LucyVerificationEngine)
-        raise NotImplementedError("No lucy-specific model created yet.")
+        model = get_default_lucy_hardware()
+    else:
+        raise NotImplementedError(f"No specific model created yet for {qpu_type}.")
 
-    return None
+    return model
 
 
 class VerificationEngine(QuantumExecutionEngine, ABC):
+
+    def __init__(self, model):
+        super().__init__(model)
```

Comment:
As it is, it should be removed. But we actually want to add add `max_circuit_duration` as a field with 90000.

Reply:
Done.